docs/setup-vendor.md

```diff
@@ -0,0 +1,51 @@
+# Getting started with Private Packagist for vendors on a Composer project
+
+Using Private Packagist, vendors let their customers install packages with Composer using a unique URL and an authentication token.
+
+As a vendor's customer, you can start using Private Packagist in either an existing or a new Composer project.
+
+## Adding a Package from a vendor to an existing Composer project
+
+Let's suppose your company bought a package named `acme-company/api` from a vendor named Acme Company. You'd like to install the package in your existing project.
```

Comment:
I am confused. Didn't we say that the documentation is for the package vendor with the goal that they have enough info to explain to their customers how they can install the packages they bought? This reads like it addresses directly the vendor customer.

Reply:
What I wanted to acheive with this, is to show the customer how they can install a package from a vendor. In this case, the customer bought a package from a vendor, and the vendor is trying to explain how to install a package. That's why it's addressing directly the vendor customer. I'm not sure if there is a better way here?

Reply:
@wissem The goal of our documentation is to inform our customers, who are vendors, what information to give to their customers. This documentation will not be read by the customers of packages themselves.

Reply:
ok, this should be adjusted now.